suppress.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
```

Comment:
does it work without the xmlns? maybe we don't have to reference that repo

Reply:
I haven't tried it, I generated this file from the dependency reprt by clicking "surpress this vulnerability"

Reply:
removing it causes it to not work locally